CMakeLists.txt

```diff
@@ -1,17 +1,41 @@
 cmake_minimum_required(VERSION 3.0.2)
 
-project(mxnet C CXX)
+
+
+if((${CMAKE_VERSION} VERSION_GREATER "3.9.0") OR (${CMAKE_VERSION} VERSION_EQUAL "3.9.0"))
+    Set(FIRST_CUDA TRUE)
+else()
+    Set(FIRST_CUDA FALSE)
+endif()
+include(cmake/Utils.cmake)
+mxnet_option(USE_CUDA             "Build with CUDA support"   ON)
+mxnet_option(USE_OLDCMAKECUDA           "Build with old cmake cuda" OFF)
```

Comment:
Can this be set automatically from the cmake version? Maybe there is a reason it should be set manually, what is it?

Reply:
This just in case backup option is used to manually switch to the old mode.